libs/localization/assets/locales/en/sheet.json

```diff
@@ -153,7 +150,9 @@
     "electro": "After triggering an <electro>Electro-related</electro> Elemental Reaction"
   },
   "bonusScaling": {
-    "skill_": "Ele. Skill Bonus Scaling"
+    "skill_": "Ele. Skill Bonus Scaling",
+    "atkInc": "ATK Inc. Bonus Scaling",
+    "percentStat": "{{percent}} {{stat}}"
```

Comment:
Is this something that can work for all languages?

Reply:
It could, in theory; turns out the issue is another, however: `[object Object]` 😭

Reply:
Yes that is the issue with nesting translation, we had this issue before. Anyways, this percent stat thing shouldnt be like this. Instead we should be using a node with 
```
{
  text: st(stat.statName),
  value: percentValue
}
```

Reply:
i'm fixing it. just took a break because i got involved with something